qaframework-bdd-tests/cypress.json

```diff
@@ -6,14 +6,14 @@
   ],
   "video": true,
   "defaultCommandTimeout": 20000,
-  "baseUrl": "https://openmrs-spa.org/openmrs/spa",
+  "baseUrl": "https://o3.openmrs.org/openmrs/spa",
   "env": {
-    "API_BASE_URL": "https://openmrs-spa.org/openmrs/ws/rest/v1",
-    "ADMIN_USERNAME": "spa_user",
```

Comment:
Meanwhile could there be a work around to resurrect this server https://openmrs-spa.org/openmrs/spa/login

Reply:
The login is not working. Btw, are we still maintaining that server?

Reply:
According to this issue (https://issues.openmrs.org/projects/O3/issues/O3-955) the openmrs-spa demo will be deprecated soon and the demo will be moved to dev3/o3 servers.